sru/modules.py

```diff
@@ -119,7 +119,9 @@ def __init__(self,
             self.activation = 'tanh'
         self.amp_recurrence_fp16 = amp_recurrence_fp16
         self.normalize_after = normalize_after
-        self.weight_c_init = weight_c_init
+        self.weight_c_init = float(1.0)
```

Comment:
Note: `1.0` is already a float:

```
In [1]: type(1.0)
Out[1]: float

In [2]: type(float(1.0))
Out[2]: float
```

Reply:
i used to get torchscript compilation errors for SRU w/o explicit float() annotation.